css/properties/user-select.json

```diff
@@ -45,17 +45,6 @@
               {
                 "prefix": "-moz-",
                 "version_added": "1"
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": "44",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.prefixes.webkit",
-                    "value_to_set": "true"
-                  }
-                ]
               }
             ],
             "firefox_android": [
```

Comment:
This one leaves Firefox for Android untouched. Some reason?

Reply:
Might've been me trying not to include the FxA changes that aren't ready yet.  Fixed!